architecture/architecture.puml

```diff
@@ -0,0 +1,39 @@
+﻿@startuml
+
+package "AV Datalogger" {
+    [Connector] as Connector
+    [Data Store] as Store
+    [Car Logger] as Log
```

Comment:
If the Car Logger only receives data from the Data Store, how is the .CSV named or stored in order to be useful? Does the Data Store also store the Car ID or something similar? Might need to follow up with Nick O regarding the message content

Reply:
Not quite sure, if we had access to a unique id / car id via the datalogger that would be nice, otherwise we would need to set that via the app. We can bring it up today and ask

Reply:
update from nick: we are to get car UID from the data logger, but this must be set via the app -> settings window